src/blueprint/operators/call.rs

```diff
@@ -36,19 +38,38 @@ pub fn compile_call(
     call: &config::Call,
     operation_type: &GraphQLOperationType,
 ) -> Valid<Expression, String> {
-    get_field_and_field_name(call, config_module).and_then(|(_field, field_name, args)| {
-        let empties: Vec<(&String, &config::Arg)> = _field
+    call.steps
+        .iter()
+        .map(|step| compile_step(field, config_module, step, operation_type))
+        .reduce(|expr, next| expr.and_then(|expr| next.map(|next| expr.and_then(next))))
+        .expect("Failed to compose call steps")
```

Comment:
use valid `from_option` and handle the error more gracefuly. Explain why it could not compose - `because steps to call can't be empty`